*** Reading remote log from gs://europe-west2-my-composer-7c66f90b-bucket/logs/dag_id=create_taxonomy_from_kpccp_to_kpdevops/run_id=scheduled__2023-01-31T14:33:33.321735+00:00/task_id=create_taxonomy_from_kpccp_to_kpdevops/attempt=4.log.
[2023-02-01, 17:10:05 UTC] {taskinstance.py:1172} INFO - Dependencies all met for <TaskInstance: create_taxonomy_from_kpccp_to_kpdevops.create_taxonomy_from_kpccp_to_kpdevops scheduled__2023-01-31T14:33:33.321735+00:00 [queued]>
[2023-02-01, 17:10:05 UTC] {taskinstance.py:1172} INFO - Dependencies all met for <TaskInstance: create_taxonomy_from_kpccp_to_kpdevops.create_taxonomy_from_kpccp_to_kpdevops scheduled__2023-01-31T14:33:33.321735+00:00 [queued]>
[2023-02-01, 17:10:05 UTC] {taskinstance.py:1369} INFO - 
--------------------------------------------------------------------------------
[2023-02-01, 17:10:05 UTC] {taskinstance.py:1370} INFO - Starting attempt 4 of 5
[2023-02-01, 17:10:05 UTC] {taskinstance.py:1371} INFO - 
--------------------------------------------------------------------------------
[2023-02-01, 17:10:05 UTC] {taskinstance.py:1390} INFO - Executing <Task(PythonOperator): create_taxonomy_from_kpccp_to_kpdevops> on 2023-01-31 14:33:33.321735+00:00
[2023-02-01, 17:10:05 UTC] {standard_task_runner.py:52} INFO - Started process 168791 to run task
[2023-02-01, 17:10:05 UTC] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'create_taxonomy_from_kpccp_to_kpdevops', 'create_taxonomy_from_kpccp_to_kpdevops', 'scheduled__2023-01-31T14:33:33.321735+00:00', '--job-id', '1578', '--raw', '--subdir', 'DAGS_FOLDER/create_taxonomy_from_kpccp_to_kpdevops.py', '--cfg-path', '/tmp/tmppxgfu5il', '--error-file', '/tmp/tmptxvqj_00']
[2023-02-01, 17:10:05 UTC] {standard_task_runner.py:80} INFO - Job 1578: Subtask create_taxonomy_from_kpccp_to_kpdevops
[2023-02-01, 17:10:06 UTC] {task_command.py:375} INFO - Running <TaskInstance: create_taxonomy_from_kpccp_to_kpdevops.create_taxonomy_from_kpccp_to_kpdevops scheduled__2023-01-31T14:33:33.321735+00:00 [running]> on host airflow-worker-rd7gn
[2023-02-01, 17:10:06 UTC] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=create_taxonomy_from_kpccp_to_kpdevops
AIRFLOW_CTX_TASK_ID=create_taxonomy_from_kpccp_to_kpdevops
AIRFLOW_CTX_EXECUTION_DATE=2023-01-31T14:33:33.321735+00:00
AIRFLOW_CTX_TRY_NUMBER=4
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-31T14:33:33.321735+00:00
[2023-02-01, 17:10:06 UTC] {logging_mixin.py:115} INFO - in create_taxonomy_from_kpccp_to_kpdevops
[2023-02-01, 17:10:06 UTC] {logging_mixin.py:115} INFO - proj id: kp-devops, location_id: eu, disp name: create_taxonomy_from_kpccp_to_kpdevops
[2023-02-01, 17:10:06 UTC] {logging_mixin.py:115} INFO - Creating taxonomy
[2023-02-01, 17:10:07 UTC] {logging_mixin.py:115} INFO - Created taxonomy projects/kp-devops/locations/eu/taxonomies/890645014463912972
[2023-02-01, 17:10:07 UTC] {python.py:173} INFO - Done. Returned value was: None
[2023-02-01, 17:10:07 UTC] {taskinstance.py:1408} INFO - Marking task as SUCCESS. dag_id=create_taxonomy_from_kpccp_to_kpdevops, task_id=create_taxonomy_from_kpccp_to_kpdevops, execution_date=20230131T143333, start_date=20230201T171005, end_date=20230201T171007
[2023-02-01, 17:10:07 UTC] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-02-01, 17:10:07 UTC] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check